packages/react/src/components/icon/icon.tsx

```diff
@@ -90,6 +91,7 @@ const iconMapping = {
     externalLink: ExternalLink,
     eye: Eye,
     eyeOff: EyeOff,
+    enso: Enso,
```

Comment:
Je repense à ça, je suis pas certain que ce soit pertinent de l'exporter dans les icônes par contre. Peut-être directement importer le SVG dans le spinner plutôt?

Reply:
C'est un point auquel j'ai pensé...
Ma réflexion est que si le spinner fait partie des icônes et que nous devons l'utiliser à l'intérieur d'un composant, _e.g._, un bouton où l'icône de gauche changerait pour l'icône de loading, nous pourrions directement le sélectionner parmi la liste d'icônes plutôt que d'utiliser le composant **_Spinner_** et possiblement alourdir le code. Le code du composant Spinner servait, en autre, à animer le SVG, maintenant que l'animation est directement dans le `SVG`, ca permettrait ça.

Pour l'instant, j'ai modifié le visuel du composant. À discuter pour la suite.

Reply:
J'aurais plus tendance à ne pas l'exporter dans les icônes pour l'instant et à le rajouter dans le futur si le besoin se présente. Un breaking change de moins! 😛 

Mais bon, c'est pas un gros concern, on peut aussi le laisser comme ça

Reply:
c'est bon. Je l'ai enlever des icons.
I thank you